### ISSUE TITLE
HTML parser removes class attr from element

### ISSUE THREAD

**Issue by m80her** [kenCode-de/react-language-selector#5]
`<a id="language" class="classes" href="#" role="button">Request a demo</a>`

When the language changes, the class attr is removed from the `<a>` tag.

**Comment by poqdavid** [comment]
Yes i noticed it but i was busy with something it must be fixed now
